docs/src/main/diataxis/doc-concept.adoc

```diff
@@ -0,0 +1,50 @@
+////
+This document is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="concept-quarkus-documentation"]
+= Quarkus documentation concepts
+include::attributes.adoc[]
+:fn-diataxis: footnote:diataxis[Procida, D. Diátaxis documentation framework. https://diataxis.fr/]
```

Comment:
Is it supported by the website? If not, I wouldn't use it.

Reply:
The footnote.. it should, but we can test that out, I guess.

I am not sure of a better way to do a citation like this that gives credit properly.

Reply:
It needs to be tested before we make it part of the first examples we have. Otherwise people will start to use it and if they are not working we will have to change them back.

And that being said, I'm not a big fan of footnotes on the web. Going back and forth to the bottom of the document is really not very practical while you can add a nice `[NOTE]`.

Reply:
Y.. but it is big for a note (yes to testing). This is not in the template that should be copied, FWIW. This is specifically the concept guide for docs.